kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableWritesSuite.scala

```diff
@@ -1321,6 +1321,369 @@ class DeltaTableWritesSuite extends DeltaTableWriteSuiteBase with ParquetSuiteBa
     }
   }
 
+  test("Test set schema on existing table") {
```

Comment:
I think we should split these tests off to their own suite -- what do you think?

Also -- @amogh-jahagirdar -- If you see the classdocs for this test suite, "Transaction commit in this suite IS REQUIRED TO use commitTransaction than .commit" -- the fact that you are _not_ doing this and things are still working fine makes me (a) think these should be in a new suite, and (b) why we need to use commitTransaction ?

cc @vkorukanti

Reply:
I think I agree with separate test suite just for organization purposes. I read through the test suits a bit more and I think the intention of saying to use `commitTransaction` instead of directly doing a `.commit` is that some test suites actually override commitTransaction e.g. the CRC tests https://github.com/delta-io/delta/blob/master/kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableWriteWithCrcSuite.scala#L40

The CRC tests invoke some post commit hooks. So if I understand right the test says to do these just so that whatever custom behavior needs to get exercised for a commit for a test suite gets invoked rather than potentially skipped/masking issues.